code/src/components/TodoInput.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { v4 as uuidv4 } from 'uuid';
+
+import { todos } from '../reducers/todos';
+import { Form, StyledLabel, InputField, InputAdd } from '../styledComponents/styled_components';
+
+// Receives listId as input
+export const TodoInput = () => {
+  const dispatch = useDispatch();
+  
+  // State for input from text-box
+  const [inputValue, setInputValue] = useState('');
+  // State for date
+  
+  const date = new Date();
+
+  // Create handle submit function to dispatch addTodo
+  const handleOnSubmit = (event) => {
+    event.preventDefault();
+    console.log(inputValue)
+
+    // This is where we will dispatch the action to save the new todo item
+    // The 'actions' are the names of the reducers we have defined. First 'actions'
+    // and then name of reducer.
+    dispatch(
+      todos.actions.addTodo({
+        description: inputValue,
+        id: uuidv4(),
+        done: false
+      })
+    );
+
+    // Clear text field after saved and component re-renders
+    setInputValue('');
+  };
+
+  return (
+    <>
+      <StyledLabel htmlFor='todo-input'>
+        date: {date.toLocaleDateString()}      
+      </StyledLabel>
+      <Form className='todo-input' onSubmit={handleOnSubmit}>
+        <InputField
+          type='text'
+          aria-label='{labelText}'
```

Comment:
I thought maybe you are getting the aria-label value from a variable called labelText, but could not find that variable anywhere. So maybe this can be made into a string?